chainstate/test-suite/src/tests/tx_verification_simulation.rs

```diff
@@ -19,20 +19,14 @@ use common::chain::{tokens::TokenIssuanceVersion, ChainstateUpgrade, Destination
 
 #[rstest]
 #[trace]
-#[case(Seed::from_entropy(), 20, 50, false)]
+#[case(Seed::from_entropy(), 20, 50)]
 #[trace]
-#[case(Seed::from_entropy(), 20, 50, true)]
-fn simulation(
-    #[case] seed: Seed,
-    #[case] max_blocks: usize,
-    #[case] max_tx_per_block: usize,
-    #[case] tx_index_enabled: bool,
-) {
+#[case(Seed::from_entropy(), 20, 50)]
```

Comment:
now that the cases are identical one can be removed

Reply:
Done.